core/trino-spi/src/main/java/io/trino/spi/security/AccessDeniedException.java

```diff
@@ -746,6 +748,21 @@ public static void denyShowCreateFunction(String functionName, String extraInfo)
         throw new AccessDeniedException(format("Cannot show create function for %s%s", functionName, formatExtraInfo(extraInfo)));
     }
 
+    public static void denySetEntityAuthorization(String ownedKind, List<String> name)
```

Comment:
Let's mark all of the old `denySet*Authorization` `@Deprecated(forRemoval = true)`